test/utils.ts

```diff
@@ -150,6 +150,7 @@ test('getOwnerAndRepo', t => {
 
 	Object.keys(ownerAndRepo).forEach(url => {
 		location.href = url;
+		// @ts-ignore
```

Comment:
Add a comment why it's ignored.
```suggestion
		// @ts-ignore Because of unicorns.
```

Reply:
Fixed error altogether by changing data structure to a Map